src/i18n/languages.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   'en-US': 'English',
+  'fr': 'French',
```

Comment:
will trigger `Unnecessarily quoted property 'fr' found.` while linting. You can use `fr` as key.
However, I will change it once merged.